pg_auth_mon.c

```diff
@@ -276,6 +288,37 @@ auth_monitor(Port *port, int status)
 	if (status == STATUS_EOF)
 		return;
 
+	/* Follow the logic in postgres/src/backend/utils/init/postinit.c:252-275 */
+	if (log_successful_authentications_guc && status == STATUS_OK) {
+		StringInfoData logmsg;
+		initStringInfo(&logmsg);
+
+		appendStringInfo(&logmsg, _("connection authorized: %s:%s user=%s database=%s"),
+							 port->remote_host, port->remote_port, port->user_name, port->database_name);
+
+		if (port->application_name != NULL)
+			appendStringInfo(&logmsg, _(" application_name=%s"),
+							 port->application_name);
+
+		appendStringInfo(&logmsg, _(" (%s:%d)"), HbaFileName, port->hba->linenumber);
+
+		if (port->ssl_in_use)
+			appendStringInfo(&logmsg, _(" SSL enabled (protocol=%s, cipher=%s, bits=%d)"),
+							 be_tls_get_version(port),
+							 be_tls_get_cipher(port),
+							 be_tls_get_cipher_bits(port));
```

Comment:
I think we should wrap this into `#ifdef USE_SSL`
```suggestion
#ifdef USE_SSL
		if (port->ssl_in_use)
			appendStringInfo(&logmsg, _(" SSL enabled (protocol=%s, cipher=%s, bits=%d)"),
							 be_tls_get_version(port),
							 be_tls_get_cipher(port),
							 be_tls_get_cipher_bits(port));
#endif
```